plugins/orchestrator/src/utils/error.ts

```diff
@@ -0,0 +1,11 @@
+import { isError } from '@backstage/errors';
```

Comment:
Can you please follow this explicit convention of naming the module `XyzUtils.ts`. The name `error.ts` is too generic and can mean different things in different contexts.